jha/src/components/enrolment/Child.vue

```diff
@@ -1102,7 +1103,9 @@ export default {
         birthDateYouthValidator: optionalValidator(birthDateYouthValidator),
         birthDateStudentValidator: optionalValidator(birthDateStudentValidator),
       },
-      personalHealthNumber: {},
+      personalHealthNumber: {
+        phnFirstDigitValidator: optionalValidator(phnFirstDigitValidator)
+      },
```

Comment:
Instead of adding the validator here, can you group this validator with the other ones for PHNs? Example: `phnValidator`. They should only be conditionally added. Same goes for account holder and spouse.

Reply:
This has also been done